uui/components/navigation/MainMenu/MainMenuDropdown.tsx

```diff
@@ -21,7 +21,7 @@ export class MainMenuDropdown extends React.Component<MainMenuDropdownProps> {
 
                     return (
                         <FocusLock returnFocus persistentFocus lockProps={ { onKeyDown: handleEscape } }>
-                            <div className={ cx(css.dropdownBody, 'uui-mainMenu-dropdown') }>
+                            <div className={ cx(css.dropdownBody, 'uui-main_menu-dropdown') }>
```

Comment:
let's avoid this change to not cause a breaking change

Reply:
this is not a breaking change, it is implemented within the framework of this RC, I just tweaked the class, it's more correct than using a camelcase